src/Fpl.Search/Searching/SearchService.cs

```diff
@@ -77,7 +77,6 @@ public async Task<SearchResult<LeagueItem>> SearchForLeague(string query, int pa
                     )
                 .Sort(sd => sd
                     .Descending(SortSpecialField.Score)
-                    .Descending(p => p.AdminCountry == countryToBoost ? 1 : 0)
```

Comment:
Hvorfor fjerne? Tror egentlig den kan være handy. Kan gjøre et kall clientside først som finner landet ditt basert på IP (kan f.eks. ha en nextjs proxy mot https://api.ipgeolocationapi.com/geolocate/{ip}), og bruke dette landet til å gi en liten bias i søket der treff har samme score. Dette vil eksempelvis gjøre det betydelig lettere å finne ice-ligaen vi er med, fremfor å paginere gjennom ørten ligaer som heter noe med "ice".

Reply:
Helt enig altså, men den kræsja søket. Fjerna en og en linje i NEST-querien før jeg fant ut at den der bare gav 400 fra ElasticSearch. Så noe er up med den. Temp fjerning til man har en fix. Det finnes også noe som heter Boosting queries, men har ikke sett nærmere på det.

Reply:
Ah, skjønner! Rart. Jeg titta litt på boosting også, husker jeg - men endte opp med det der. Husker ikke om det var fordi jeg ikke fikk det til å funke eller om det ga litt for feil bias. Skal se om jeg kan få fiksa det.